Discord.js

```diff
@@ -1441,6 +1441,15 @@ bot.IPC.on("restart", async (msg, callback) => {
 	}
 });
 
+bot.IPC.on("updateConfigJSON", async (_, callback) => {
```

Comment:
`reloadConfig` as the event name would make more sense, also, you can still call the first parameter `msg`, it would be less confusing for contributors.

Reply:
The first param is not needed, hence why it's called `_`. It's ignored

Reply:
I fully understand the usage, I just disagree with it. It isn't consistent compared to the other IPC callbacks and naming it semanticly just makes it look nicer too.